src/Tizen.TV.Extension.UIControls.Forms/TVESPlayer.cs

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using Xamarin.Forms;
+using Tizen.TV.UIControls.Forms;
+
+namespace Tizen.TV.Extension.UIControls.Forms
+{
+    /// <summary>
+    /// TVESMediaPlayer provieds the essential components to play the element stream.
+    /// </summary>
+    public class TVESPlayer : MediaPlayer
+    {
+        ITVESPlayer _esImpl;
+
+        /// <summary>
+        /// Initializes a new instance of the TVESPlayer class.
+        /// </summary>
+        public TVESPlayer() : base()
+        {
+            if (_impl == null) {
+                _impl = CreateMediaPlayerImpl();
+            }
+            _esImpl = _impl as ITVESPlayer;
+            _esImpl.ResourceConflicted += SendResourceConflicted;
+            _esImpl.BufferStatusChanged += SendBufferStatusChanged;
+            _esImpl.EOSEmitted += SendEOSEmitted;
+            _esImpl.ErrorOccurred += SendErrorOccurred;
+            _esImpl.StreamReady += SendStreamReady;
+            _esImpl.SeekReady += SendSeekReady;
+        }
+
+        protected override IPlatformMediaPlayer CreateMediaPlayerImpl()
+        {
+            return DependencyService.Get<ITVESPlayer>(fetchTarget: DependencyFetchTarget.NewInstance) as IPlatformMediaPlayer;
+        }
+
+        /// <summary>
+        /// Stream information for es contents of audio stream
+        /// </summary>
+        public AudioStreamInfo AudioStreamInfo
+        {
+            set 
+            {
+                _esImpl.AudioStreamInfo = value;
+            }
+        }
+
+        /// <summary>
+        /// Stream information for es contents of video stream
+        /// </summary>
+        public VideoStreamInfo VideoStreamInfo
+        {
+            set
+            {
+                _esImpl.VideoStreamInfo = value;
+            }
+        }
+
+        /// <summary>
+        /// Get the time for playing of player
+        /// </summary>
+        public TimeSpan PlayingTime

```

Comment:
What is diffrent from `Position` on MediaPlayer

Reply:
Actually, `PlayingTime` is same with `Position` except of `Type`